### PR TITLE
feat: add mass example

### PR DESCRIPTION
**Pull request by lpatiny**
close https://github.com/zakodium/react-plot/issues/74

### CODE REVIEW

File: stories/spectra/mass.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, LineSeries, Plot } from '../../src';
+import data from '../data/infrared.json';
+import { DEFAULT_CONFIG } from '../utils';
+
+export default {
+  title: 'Experimental spectra/Infrared',
```
Comment (maasencioh) [inline stories/spectra/mass.stories.tsx:8]: Isn't it mass?

File: stories/spectra/mass.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+} as Meta;
+
+export function InfraredExample() {
```
Comment (maasencioh) [inline stories/spectra/mass.stories.tsx:11]: same

File: stories/spectra/infrared.stories.tsx [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 
-import { Axis, LineSeries, Plot } from '../../src';
-import data from '../data/infrared.json';
+import { Axis, BarSeries, Plot } from '../../src';
+import data from '../data/mass.json';
 import { DEFAULT_CONFIG } from '../utils';
 
 export default {
-  title: 'Experimental spectra/Infrared',
+  title: 'Experimental spectra/Mass',
```
Comment (maasencioh) [inline stories/spectra/infrared.stories.tsx:8]: The file is called infrared, I think that you had a confussion

**Comment by targos** [comment]
commit message should start with `docs:`

File: stories/spectra/mass.stories.tsx (hunk 1 above)
Comment (lpatiny) [reply stories/spectra/mass.stories.tsx:8]: Thanks. A mistake I often do, copy paste a file and change the original file rather than the copy.

**Pull request merged**